_build/data/transport.core.system_settings.php

```diff
@@ -567,7 +567,7 @@
 $settings['feed_modx_security_enabled']= $xpdo->newObject('modSystemSetting');
 $settings['feed_modx_security_enabled']->fromArray(array (
   'key' => 'feed_modx_security_enabled',
-  'value' => '1',
+  'value' => '0',
```

Comment:
This is me and git only slowly getting friends. Sorry. :-(
Actually I went back from "off/0" to "on/1", so the file is as was before I made changes. I reverted.